evm/src/trace/utils.rs

```diff
@@ -69,5 +87,11 @@ pub(crate) fn decode_cheatcode_outputs(func: &Function, _data: &[u8]) -> Option<
         // redacts derived private key
         return Some("<pk>".to_string())
     }
+    if func.name == "parseJson" && verbosity != 5 {
+        return Some("<encoded JSON value>".to_string())
+    }
```

Comment:
TBH I'm not a big fan of the change in this PR. @odyslam I know you have some huge JSON files so this makes sense for you, but it seems too specific for that use case—I often use small JSON files with a handful of keys, and having to set verbosity 5 to see the inputs in traces isn't great for that use case.

Personally I would prefer this to be "show the text in traces if verbosity is 5 OR if test/script fails and a trace is shown (verbosity >=3)", but I see why you wouldn't want that. 

Seems the best way to support this is to start on https://github.com/foundry-rs/foundry/issues/3390 and implement this as a config option, but open to other ideas

Reply:
And to clarify, I don't think implementing this PR makes the need for #3390 urgent—I can live with this change, it's just not optimal IMO 🙂

Reply:
We should def work on #3390. 

For us, foundry became utterly impossible to work with (about 90K lines of tracing output), because we read/write multiple times from a json config file. I assumed that it would be a common pain.

Reply:
> about 90K lines of tracing output)

Whew haha that is wild! Def makes sense why you need this. And yea just wanted to convey I think this is totally fine as a short term solution to quickly resolve your pain points but we prob shouldn't keep it as the long term solution. Sounds like we're on the same page here though, thanks 👌